airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/components.py

```diff
@@ -42,21 +46,33 @@ def get_headers(self):
     @backoff.on_exception(
         backoff.expo,
         DefaultBackoffException,
+        max_tries=2,
         on_backoff=lambda details: logger.info(
             f"Caught retryable error after {details['tries']} tries. Waiting {details['wait']} seconds then retrying..."
         ),
         max_time=300,
     )
     def _get_refresh_access_token_response(self):
         try:
-            response = requests.request(
-                method="POST", url=self.get_token_refresh_endpoint(), data=self.build_refresh_request_body(), headers=self.get_headers()
-            )
+            request_url = self.get_token_refresh_endpoint()
```

Comment:
@Jgerardopine do you know why this needs to be a custom component? It's not obvious to me why this can't be a session or oauth authenticator since this looks pretty standard

Reply:
This was an existing working component which I did not change. But I can certainly try to migrate to a std manifest component.

Reply:
@girarda  I will migrate this component after the certification as I need this for a couple of tests I need to run for a different project.